### PR TITLE
Tune limits

### PR DESCRIPTION
**Pull request by xzfc**
## Description

Tune limits on containers introduced in #2188.

New limits:

|Container image  |Memory|   CPU|
|-----------------|-----:|-----:|
|`nsm-monitor`    |`15Mi`|`100m`|
|`coredns`        |`15Mi`|`100m`|
|`nsm-init`       |`20Mi`|`200m`|
|`nsm-dns-init`   |`15Mi`|`200m`|

Changes made: tuned down CPU limit for `nsm-init` and `nsm-dns-init` from `500m` to `200m`.

## Motivation and Context

The CPU limits were so high, so k8s was occasionally unable to allocate CPUs on the CI.
Evidence: the following string found in these CI logs.
```
\"reason\":\"Unschedulable\",\"message\":\"0/2 nodes are available: 1 Insufficient cpu
```
* https://app.circleci.com/pipelines/github/networkservicemesh/networkservicemesh/9115/workflows/ef506fb2-e44a-488f-b9de-81a193ded542/jobs/122567
* https://app.circleci.com/pipelines/github/networkservicemesh/networkservicemesh/9115/workflows/881f2b42-45a0-442c-b93f-adb6d847b217/jobs/122557
* https://app.circleci.com/pipelines/github/networkservicemesh/networkservicemesh/9113/workflows/a68c2799-94c0-419c-97d7-82e43140bb2e/jobs/122532
* https://app.circleci.com/pipelines/github/networkservicemesh/networkservicemesh/9113/workflows/a306efb1-d04e-481d-b4b9-f413982f70ba/jobs/122527

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by edwarnicke** [comment]
@xzfc Glad to see you tuning these limits... would be good to get some comments in the code talking through the reasoning for the tuning for the next person :)

File: test/kubetest/k8s.go [hunk 1]
```diff
@@ -378,9 +378,20 @@ func NewK8sWithClientset(clientset kubernetes.Interface, namespace string) *K8s
 	return r
 }
 
+func runShell(cmd string) {
```
Comment (denis-tingajkin) [inline test/kubetest/k8s.go:381]: Can you just add

`kubectl describe nodes`  into https://github.com/networkservicemesh/networkservicemesh/blob/master/.cloudtest/executions/single-cluster-tests.yaml#L17?

**Review by denis-tingajkin — COMMENTED** [review]
As I can see this PR fix fails on azure clusters with DeploymentOrder* tests. 
Why this PR still draft?

**Comment by xzfc** [comment]
> Why this PR still draft?

I was going to remove debug stuff (`runShell("kubectl describe nodes")`) before merging.

File: test/kubetest/k8s.go (hunk 1 above)
Comment (xzfc) [reply test/kubetest/k8s.go]: I'll remove it as it is no longer needed.

**Comment by denis-tingajkin** [comment]
Here is a failed one test.
Looks like it is a new issue and it is not related to this PR.
The issue to track status: https://github.com/networkservicemesh/networkservicemesh/issues/2198


We'll fix the test failing root cause in separate PR.

**Review by denis-tingajkin — APPROVED** [review]

**Pull request merged**